src/testRunner/externalCompileRunner.ts

```diff
@@ -78,7 +78,7 @@ namespace Harness {
                         exec("npm", ["i", "--ignore-scripts"], { cwd, timeout: timeout / 2 }); // NPM shouldn't take the entire timeout - if it takes a long time, it should be terminated and we should log the failure
                     }
                     const args = [path.join(IO.getWorkspaceRoot(), "built/local/tsc.js")];
-                    if (types) {
+                    if (types?.length) {
                         args.push("--types", types.join(","));

```

Comment:
this is not correct; `args` needs to include `--types` even if the list is empty. `npm install` is OK to skip though.

I think this will make the diffs in the baselines go away, unfortunately. The real fix for those projects is to add `types` entries to their tsconfigs.